### PR TITLE
New power quest

### PR DESCRIPTION
**Pull request by kjhenner**
Initial version of the new Power of Puppet quest.

### CODE REVIEW

File: Quest_Guide/quests/power_of_puppet.md [hunk 1]
```diff
@@ -11,118 +11,137 @@ layout: default
...
-get there when the time is right!
+In this quest you will use the Puppet Enterprise (PE) console to set up Graphite,
+an open-source graphing tool that lets you easily visualize the state of your
+infrastructure. Graphite, like Puppet, helps span the gap between nuts-and-bolts and
+the big-picture, so it's a nice example to get you started on your path to Puppet mastery.
 
-When you're ready to get started, type the following command:
+For the sake of simplicity, we've already written out the HTML for a simple dashboard.
+Take a look [here](http://localhost:90/graphite_dashboard.html). Of course, without
+Graphite running, there's not much to display. Go ahead and take
```
Comment (joshsamuelson) [inline Quest_Guide/quests/power_of_puppet.md]: Looks like a missing word or two.  "Go head and take a look?"

File: Quest_Guide/quests/power_of_puppet.md [hunk 2]
```diff
@@ -11,118 +11,137 @@ layout: default
...
-	
+    puppet module install dwerder-graphite
+    
 {% aside Offline? %}
 If you don't have internet access, run the following terminal commands to use
-cached versions of all the modules used in the quests:
+cached versions of all the modules reqiuired for quests in this guide:
 
     cd /usr/src/forge/
     for m in `ls`; do puppet module install $m --ignore-dependencies ; done
```
Comment (joshsamuelson) [inline Quest_Guide/quests/power_of_puppet.md]: I think I gave you this script, but I recently discovered that you can just use this:
for m in /usr/src/forge/*; do puppet module install $m --ignore-dependencies; done

File: Quest_Guide/quests/power_of_puppet.md [hunk 3]
```diff
@@ -175,197 +195,68 @@ interface to commit your change.
...
 necessary to bring the node into the line with the state described by the
 catalog.
 
-{% task 3 %}
+{% task 4 %}
 ---
 - execute: |
-    curl -i -k --cacert /etc/puppetlabs/puppet/ssl/ca/ca_crt.pem --key /etc/puppetlabs/puppet/ssl/private_keys/learning.puppetlabs.vm.pem --cert /etc/puppetlabs/puppet/ssl/certs/learning.puppetlabs.vm.pem -H "Content-Type: application/json" -X POST -d '{"name":"Learning VM", "environment":"production", "parent":"00000000-0000-4000-8000-000000000000", "classes":{"lvmguide" : {} },  "rule":["or", ["=", "name", "learning.puppetlabs.vm"]]}' https://localhost:4433/classifier-api/v1/groups
+    curl -i -k --cacert /etc/puppetlabs/puppet/ssl/ca/ca_crt.pem --key /etc/puppetlabs/puppet/ssl/private_keys/learning.puppetlabs.vm.pem --cert /etc/puppetlabs/puppet/ssl/certs/learning.puppetlabs.vm.pem -H "Content-Type: application/json" -X POST -d '{"name":"Learning VM", "environment":"production", "parent":"00000000-0000-4000-8000-000000000000", "classes":{"graphite" : {"gr_apache_port" : "90"} },  "rule":["or", ["=", "name", "learning.puppetlabs.vm"]]}' https://localhost:4433/classifier-api/v1/groups
```
Comment (joshsamuelson) [inline Quest_Guide/quests/power_of_puppet.md]: What's this about?  Isn't this duplicating what we just instructed them to do?

Comment (kjhenner) [inline Quest_Guide/quests/power_of_puppet.md]: Yeah. This is the API call for the solver tool to match the console steps.

**Pull request merged**